lib/vanagon/project.rb

```diff
@@ -683,7 +683,7 @@ def publish_yaml_settings(platform)
       end
 
       File.open("#{filepath}.sha1", 'w') do |f|
-        f.write(system("#{platform.shasum} #{filepath}", err: File::NULL))
+        f.write(%x(shasum #{filepath}))
```

Comment:
Ah, I could just do this with `Digest::SHA1`, since this is local - update coming.